index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link rel="stylesheet" href="style.css">     
+        <title>Personal-Portfolio</title>
+    </head>
+  
+    <body>
+          <!--menu-->
+        <nav id="navbar">
+            <ul class="links">
+                <li>
+                    <a href="#welcome-section">About</a>
+                </li>
+                <li>
+                    <a href="#projects">Work</a>
+                </li>
+                <li>
+                    <a href="#Contact">Contact Us</a>
+                </li>
+            </ul>
+        </nav>
+      <!--welcome-->
+        <section id="welcome-section" >
+            <div class="welcome-wrapper">
+                <h1>My name is Olipliche, I am a <a href="https://github.com/OLIPLICHE">softare developer</a></h1>
+                <p>Also, I am a product of microverse school in US. I stay in Ghana</p>
+            </div>
+        </section>
+      <!--Projects-->
+        <section id="projects" class="projects">
+                <h2>Our Projects</h2>
+            <div class="project">
+                <a href="#"  class="project-tile">
+                <img src="https://devpractical.com/public/tribute-page-screenshot.jpg" alt="TechCrunch-Poject">
+                <h3>Tribute Page</h3>
+                </a>
+ 
+               <a href="#"   class="project-tile">
+                 
+                     <img  src="https://i.ytimg.com/vi/KUOi3gl-D7s/maxresdefault.jpg" alt="mashable-project">
+                     <h3>Products Landing Page</h3>
+                     
+                 
+             </a>
+ 
+             <a href="#"   class="project-tile">
+                 
+                     <img src="https://uploads-ssl.webflow.com/6080270252ce5fe66cb0c44a/6092bf855db627493b1d9441_fcc-survey-1.jpg" alt="tnw">
+                     <h3>Survey Page</h3>
+                     
+                 
+             </a>
```

Comment:
- [ ] Make sure not to add extra blank lines or indentations. Check the comment above for more details.